Deepanshu_Balani/project1 profile/profile_new.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>myprofile</title>
+    <link rel="stylesheet" type="text/CSS" href="./profiles.css" />
+    <link
+      rel="stylesheet"
+      type="text/CSS"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      <link
+      href="https://fonts.googleapis.com/css?family=Gelasio&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width,initial-state=1"/>
+<link href="css/bootstrap.css "rel="stylesheet"/>
+<script src="js/jquery-3.2.1.min.js"></script>
+<script src="js/bootstrap.js"></script>
+<style type="text/css">
+    <style>
+      h2:hover ul {
+        display: block;
+        background-color: turquoise;
+        
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container-fluid back">
+      
+         <div class="row">
+            <div class="col-lg-1 col-md-4 col-sm-12 col-xs-12"></div>
+              <div class="row">
+                  <h2 id="headings">Education</h2>
+       
+                         <ul class="list">
+                            <li>graduated from BPIT in Electrical and Electronics Engineering</li>
+                             <li>passed class XII with aggregate of 85.8%</li>
+                              <li>passed class X with 9.4 CGPA</li>
+                           </ul>
+                  </div>
+             <div class="row">
+        <h2 id="headings">Achievements</h2>
+        <ul class="list">
+          <li>passed class X and XII with distinction</li>
+          <li>SSB recommended candidate</li>
+          <li>Recieved many prizes for quizzes,debates and declamations</li>
+        </ul>
+      </div>
+    
+  
+   
+      
+        <div class=" col-md-4 col-sm-12 col-xs-12"></div>
+
+      <div class="pro">
+        <img src="./my image.jpg" />
```

Comment:
Improper folder structure
this should under /images folder within the assets folder under project folder

Reply:
made proper folder structure